src/krux/pages/login.py

```diff
@@ -132,6 +130,8 @@ def _new_key_from_die(self, states, min_rolls, max_rolls):
                     roll = self.capture_from_keypad(
                         t("Roll %d") % (i + 1), states, lambda r: r
                     )
+                    if roll == "Esc":
```

Comment:
This should be `display.ESC` so translated strings will also match (if different)